src/style.css

```diff
@@ -1 +1,88 @@
 /* Styles go here */
+html:root {
+  font-family: "Roboto", "Arial", sans-serif;
+  --color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+}
+
+.home {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  margin: 0 auto;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  align-content: center;
+  padding: 0;
+  margin: 0;
+  text-align: center;
+}
+
+.nav__box {
+  margin-right: 20px;
+}
+
+.nav__box:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  line-height: 60px;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  font-family: inherit;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.is-active {
+  color: var(--color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  height: 4px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: block;
+  background: var(--color);
+  border-radius: 8px;
+}
+
+.nav__link:hover {
+  color: var(--color);
+}
+
+@media (max-width: 1024px) {
```

Comment:
In task you also have case with 1200px
<img width="844" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/6568745/7d1ca3b5-f1b2-4240-9000-ec2785de6be7">